prisma/migrations/20210529200211_add_tables_for_nextjs_auth/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "Account" (
+    "id" TEXT NOT NULL,
+    "userId" TEXT NOT NULL,
+    "providerType" TEXT NOT NULL,
+    "providerId" TEXT NOT NULL,
+    "providerAccountId" TEXT NOT NULL,
+    "refreshToken" TEXT,
```

Comment:
For any TEXT fields, there doesn't seem to be too much of a difference between TEXT and VARCHAR in postgres. The only diff is that in VARCHAR we can limit the size. Do you think this additional safety of having a max limit for varchar would be beneficial?

Reply:
These migration files are created from the `schema.prisma` file. If we specify the limit in that file - it'll reflect in the migration file as well.